ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.5__4.3.yaml

```diff
@@ -0,0 +1,43 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: "4.3"
+    namespace: ocp
+    tag: base
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+images:
+- context_dir: serving/ingress
+  from: base
+  to: knative-openshift-ingress
+- context_dir: knative-operator
+  from: base
+  to: knative-operator
+promotion:
+  name: openshift-serverless-nightly
+  namespace: openshift
+resources:
+  '*':
+    limits: {}
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  cluster: https://api.ci.openshift.org
+  name: "4.3"
+  namespace: ocp
+tests:
+- as: e2e-aws-ocp-43
+  commands: make test-e2e
+  openshift_installer_src:
+    cluster_profile: aws
+- as: e2e-aws-ocp-43-continuous
+  commands: make test-e2e
+  cron: 0 */6 * * *
```

Comment:
Why do you add this periodic job? Isn't it sufficient to have the periodic job on the master branch only? This is a release branch that is usually active for about a week or two so I'm curious what's the goal.